docs/milo/manual/chapters/30_hardware_reference.md

```diff
@@ -52,55 +52,55 @@ A PC standard fan mounting screw. Exclusively used to hold the system cooling fa
 
 ![Heat-set Insert wireframe](../img/hardware/heat_insert.png){: .shadow}
 
-Heat-set inserts with a soldering tip so that they melt the plastic when installed. As the plastic cools, it solidifies around the knurls and ridges on the insert for excellent resistance to both torque and pull-out.
+Heat-set inserts with a soldering tip so that they melt the plastic when installed. As the plastic cools, it solidifies around the knurls and ridges on the insert for excellent resistance to torque and pull-out.
```

Comment:
This block doesn't read right. I think it's trying to say "Heat the inserts with a soldering tip so they melt the plastic when installed" but it doesn't really work with "Heat-set inserts".